### ISSUE TITLE
[BUG]: Docker images hlm-backend:3.2.4 and hlm-proxy:1.1.0 not working on arm64 arch

### ISSUE THREAD

**Issue by AdamOakman** [healenium/healenium#6]
### Describe the bug

Looks like below images in recent versions cannot be run in containers after attempting to run `docker compose up`:

healenium/hlm-backend:3.2.4
healenium/hlm-proxy:1.1.0

on Apple M1 chip in MacOS (arm64 arch). Looks like amd64 emulation fails. Is there any way I can make it work on such machine?

### How to reproduce the issue

```shell
Docker Desktop Version 4.12.0 (85629)
MacOS (Apple M1 chip, arm64)
```

### Logs appeared during using Healenium

```shell
No logs from above containers, unable to run any test
```

### Expected behavior

Healenium should work on arm64 architecture

### Actual behavior

amd64 emulation on arm64 fails for images versions provided in docker-compose.yml file

### Healenium Web version

None

### Healenium Backend version

3.2.4

### Platform

JavaScript

**Comment by ElenaStepuro** [comment]
Hello @AdamOakman please, update Docker Desktop version. It successfully works with Docker Desktop version 20+ on MacOS M1

**Comment by AdamOakman** [comment]
The docker engine I have currently installed is actually 20.10.17 (up to date as far as I checked)

**Comment by Jasper-le** [comment]
It cannot work on Mac M1 Pro either

**Comment by Alex-Reif** [comment]
Hi @Jasper-le , @AdamOakman , fixed in hlm-proxy 1.3.3 and hlm-backend 3.3.3
